src/StringSorterAscTest.java

```diff
@@ -0,0 +1,28 @@
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
(Just transferring my comment to this new PR) You can use the Assert library provided by Junit to avoid overhead but using the Jupiter API is correct too and this is a very well-written test, awesome job!